control-plane/catalog/to-consul/resource_test.go

```diff
@@ -2154,11 +2154,22 @@ func createEndpointSlice(t *testing.T, client *fake.Clientset, serviceName strin
 					Conditions: discoveryv1.EndpointConditions{
 						Ready:       pointer.Bool(true),
 						Serving:     pointer.Bool(true),
-						Terminating: pointer.Bool(false),
+						Terminating: nil,
 					},
 					NodeName: &node2,
 					Zone:     pointer.String("us-west-2b"),
 				},
+				{
+					Addresses: []string{"3.3.3.3"},
+					Conditions: discoveryv1.EndpointConditions{
+						Ready:       pointer.Bool(false),
+						Serving:     pointer.Bool(false),
+						Terminating: pointer.Bool(true),
+					},
+					TargetRef: &targetRef,
+					NodeName:  &node1,
+					Zone:      pointer.String("us-west-2a"),
+				},
```

Comment:
Can you add one more address with `Ready: pointer.Bool(true),`?  Just to cover all cases.

Reply:
Sure I'll add more but could you elaborate please? I think all 3 possible values for `Ready` should be covered here and `Ready: pointer.Bool(true)` is there on [line 2155](https://github.com/hashicorp/consul-k8s/pull/3874/files#diff-49e18150b63f1fa627a95f2f6c502f9f554a15ed8c9ac740787e3b7fa1ede710R2155).

Reply:
Ah you're right, I didn't see that in the diff initially.

Reply:
No worries! That line was already there